onnxruntime/core/mlas/lib/qgemm.h

```diff
@@ -362,7 +363,7 @@ Return Value:
                 int32_t* RowSums = RowSumBuffer;
                 size_t RowsRemaining = CountM;
 
-                bool ZeroMode = (k == 0);
+                bool ZeroMode = (k == 0) && !IsAccumulateMode;
```

Comment:
this will have performance impact. zero mode is faster.

Reply:
This is to add C. You need to do the operation any way, either here or in a post processor.